Makefile

```diff
@@ -156,7 +156,7 @@ publish-local: build
 	cp -r dist/* ../packs/node_modules/@codahq/packs-sdk/dist/
 
 .PHONY: validate-no-changes
-validate-no-changes: compile
+validate-no-changes: compile docs
```

Comment:
this worked without doing the whole `build` step so we won't have to re-lint but lmk if there was another reason you wanted to do build @jonathan-codaio